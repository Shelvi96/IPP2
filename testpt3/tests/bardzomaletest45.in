NEW ID

813>6221120
:850:14>093807:4
96>2909;2
583;7;471>3;
062515126>08
09464>2781:40453
;81216>6955
55>39;7616634
384;3>78903064
3530>:
1986331;5>0183
34677;>3480;829:3
7992;>;81188
;08>;
36>;637856
;5>1;42:4
3035>821;14
@70569:049992854;
@;2178872167909412;2436154419
@4;474371:1648
@590:;;037:9065545:719;48421;;
@78;76:;:6603:707
@99:;
@17662
@91949037
@298093243:7
@794:;;4514665:351278
@889;;5715802
@:137:934302;98144
@44247;;04;173;0::3717;12::2376
@;22722:2:13987:
@512705027;4;:2;9929;
@71699454649;;8;
