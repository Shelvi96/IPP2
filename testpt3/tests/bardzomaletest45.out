2101
4943079142262910
2
400798848719536128
784
0
0
0
0
40760786
0
22010
35207697398781069
136
9566100
64
